core/services/headtracker/types/types.go

```diff
@@ -0,0 +1,36 @@
+package types
+
+import (
+	"context"
+
+	"github.com/smartcontractkit/chainlink/core/store/models"
+)
+
+// HeadTrackable represents any object that wishes to respond to ethereum events,
+// after being attached to HeadTracker.
+//go:generate mockery --name HeadTrackable --output ../internal/mocks/ --case=underscore
+type HeadTrackable interface {
```

Comment:
This seems identical to HeadBroadcastable?

Reply:
ah yes i was just talking with Blaz about removing one